src/classes/ADDR_Addresses_TEST.cls

```diff
@@ -998,6 +998,254 @@ public with sharing class ADDR_Addresses_TEST {
 
     /*********************************************************************************************************
     @description
+        Update Contact's mailing address (with State & Country Picklists enabled) and mark it as override,
+        to cause creation of a new non-default Address record with the State & Country values set as expected.
+    verify:
+        Contact gets new address
+        HH sticks with default Address
+        New Address created with State value set as expected
+    **********************************************************************************************************/
+    static testMethod void testContactAddressOverrideForStateChangeWithStateAndCountryPicklistEnabled() {
+        if (!ADDR_Addresses_TDTM.isStateCountryPicklistsEnabled) {
+            return;
+        }
+
+        if (strTestOnly != '*' && strTestOnly != 'testContactAddressMarkOverrideWithStateAndCountryPicklistsEnabled') {
+            return;
+        }
+
+        Integer hhSize = 1;
+        Integer contactSize = 1;
+
+        // This creates a default Address for each HH
+        createHHTestData(hhSize, contactSize);
+        System.assertEquals(1, getAddresses().size());
+
+        // Modify the Contact addresses directly, and set override
+        Contact con = listConT[0];
+        con.put('MailingStateCode', 'CA');
+        con.put('MailingCountryCode', 'US');
+        con.is_Address_Override__c = true;
+
+        Test.startTest();
+        update con;
+        Test.stopTest();
+
+        // Verify a new Address record was created
+        List<Address__c> listAddr = getAddresses();
+        System.assertEquals(2, listAddr.size());
+
+        // Populate a Map of AccountId to Address record
+        Map<Id, Address__c> accountIdToAddress = new Map<Id, Address__c>();
+        for(Address__c eachAddress : listAddr) {
```

Comment:
Should we add a space after each `for`, `if` as in the rest of the code?

Reply:
Spaces added. Thanks!